src/Extensions/Hosting.Services/Internal/ServiceInitializationEventSource.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	/// <summary>
+	/// Service Fabric event source
+	/// </summary>
+	[EventSource(Name = "Microsoft-OMEX-Logs-ServiceInitialization")] //Breaking Change: event source renamed
```

Comment:
Do we want to commit this comment, or just notify stakeholders?

Reply:
I would like to commit them and remove in a separate PR after verifying that all of them are properly handled during log aggregation.